### ISSUE TITLE
Description is not rendered for array type

### ISSUE THREAD

**Issue by hungbie** [eclipsesource/jsonforms#2122]
### Describe the bug

Normally, a description will be render for the field if they have `description` in their schema.
However, for field of type `array`, the `description` field is not being rendered.

### Expected behavior

`description` being rendered.

### Steps to reproduce the issue

Example code:
```typescript
import React from 'react';
import { JsonForms } from '@jsonforms/react';
import {
  materialRenderers,
  materialCells,
} from '@jsonforms/material-renderers';

export default function ExampleJsonForm() {
  const schema = {
    type: 'object',
    description: 'This is a description of the object.',
    properties: {
      comments: {
        type: 'array',
        description: 'This is a description of the comments array.',
        items: {
          type: 'object',
          properties: {
            unfocusedDescription: {
              type: 'string',
              description: 'This description is shown even when the control is not focused',
            },
          },
        },
      },
      unfocusedDescription: {
        type: 'string',
        description: 'This description is shown even when the control is not focused',
      },
    },
  };
  const uischema = {
    type: 'VerticalLayout',
    elements: [
      {
        type: 'Control',
        scope: '#/properties/comments',
        options: {
          showUnfocusedDescription: true,
        },
      },
      {
        type: 'Control',
        scope: '#/properties/unfocusedDescription',
        options: {
          showUnfocusedDescription: true,
        },
      },
    ],
  };
  const data = {
  };
  return (
      <JsonForms
        schema={schema}
        uischema={uischema}
        data={data}
        renderers={materialRenderers}
        cells={materialCells}
      />
  );
}
```

### Screenshots

![Screenshot 2023-03-31 at 4 31 36 PM](https://user-images.githubusercontent.com/25290547/229068582-3faa94a1-4a23-4252-8105-037c6c2804c3.png)

### In which browser are you experiencing the issue?

Google Chrome Version 111.0.5563.110 (Official Build) (x86_64)

### Which Version of JSON Forms are you using?

v3.0.0

### Framework

React

### RendererSet

Material

**Comment by hungbie** [comment]
hope someone can take a look at this

**Comment by sdirix** [comment]
Hi @hungbie, our array renderer does not print the description anywhere. I could imagine to support it as an overlay to the array label, similar to the boolean controls. Would you like to contribute this feature?

**Comment by hungbie** [comment]
@sdirix sure. Happy to contribute, can you give me some pointers to where should I put this enhancement in?

**Comment by SaSteffen** [comment]
@hungbie Did you come around to doing this? I'd jump in otherwise.
@sdirix :
This is done via `FormHelperText` like [here](https://github.com/eclipsesource/jsonforms/blob/662d5985a6bbf1905f40560ea9ccfe7357771330/packages/material-renderers/src/controls/MaterialBooleanControl.tsx#L134C8-L134C22), correct?

**Comment by sdirix** [comment]
I'm not sure the `FormHelperText` is the best choice here, as the title of the array is not really a form or input. Feel free to try it, however I guess we will need to manually create a `Typography` element which looks good.

Alternatively we could just render it as a tooltip as we can't to any proper focus management here.

**Comment by hungbie** [comment]
> @hungbie Did you come around to doing this? I'd jump in otherwise. @sdirix : This is done via `FormHelperText` like [here](https://github.com/eclipsesource/jsonforms/blob/662d5985a6bbf1905f40560ea9ccfe7357771330/packages/material-renderers/src/controls/MaterialBooleanControl.tsx#L134C8-L134C22), correct?

Hi! I didn’t find the time to work on this but really appreciate if this can be done! Thank you!

**Comment by SaSteffen** [comment]
Made a minimal change. This would work for my use case.

**Comment by sdirix** [comment]
Implemented with #2227. Thanks!
